mono/mini/method-to-ir.c

```diff
@@ -7049,7 +7049,7 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 
 	ins_flag = 0;
 	start_new_bblock = 0;
-	MonoOpcodeEnum il_op = MonoOpcodeEnum_Invalid;
+	MonoOpcodeEnum il_op; il_op = MonoOpcodeEnum_Invalid;
```

Comment:
is that really needed? if yes, please insert a new line.

Reply:
Yes & ok.